app/src/pages/inside/common/defectTypeItemML/defectTypeItemML.jsx

```diff
@@ -21,21 +21,30 @@ import styles from './defectTypeItemML.scss';
 
 const cx = classNames.bind(styles);
 
-export const DefectTypeItemML = ({ defectType, onClick, isSelected, className }) => {
+export const DefectTypeItemML = ({
+  defectType,
+  onClick,
+  isSelected,
+  className,
+  isShownLess: isShowMore,
```

Comment:
See comment https://github.com/reportportal/service-ui/pull/2703#discussion_r637920438

You can call component props according to its meaning **for this component**.
If the component should expand some content (should **show more**), then it should receive the prop `showMore`.